winevtlog_bench/ansible/windows.yml

```diff
@@ -4,11 +4,22 @@
     td_agent_version: "3.8.0-0"
     eventlog_bencher_version: "v0.4.1.0"
   tasks:
+    - name: Copy local modified version for testing
+      win_copy:
+        src: ./config/td-agent-4.0.50-x64.msi
```

Comment:
Hardcoded version is not desired on this repository.
Could you use a variable which is defined in `vars` and use it?

Reply:
Yes, it is better to do so.